99-context.Rmd

```diff
@@ -0,0 +1,202 @@
+
+Then the `.md` file is processed by [Pandoc](https://pandoc.org), a multipurpose tool designed to convert files from one markup language to another. It takes any parameters specified within the YAML frontmatter of the document (e.g., `title`, `author`, and `date`) to convert the document to the output format specified in the `output` parameter (such as `html_document` for HTML output).
+
+If the output format is PDF, there is an additional layer of processing, as Pandoc will convert the intermediate `.md` file into an intermediate `.tex` file. This file is then processed by [LaTeX](https://www.latex-project.org) to form the final PDF document. As we mentioned in Section \@ref(install-latex), the **rmarkdown** package calls the `latexmk()` function in the **tinytex** package [@R-tinytex], which in turn calls LaTeX to compile `.tex` to `.pdf`.
+
+In short, `rmarkdown::render()` = `knitr::knit()` + Pandoc (+ LaTeX for PDF output only).
+
+Robin Linacre has written a nice summary of the relationship between R Markdown, **knitr**, and Pandoc at https://stackoverflow.com/q/40563479/559676, which contains more technical details than the above overview.
+
+Note that not all R Markdown documents are eventually compiled through Pandoc. The intermediate `.md` file could be compiled by other Markdown renderers. Below are two examples:
+
+- The **xaringan** package [@R-xaringan] passes the `.md` output to a JavaScript library, which renders the Markdown content in the web browser.
+
+- The **blogdown** package [@R-blogdown] supports the `.Rmarkdown` document format, which is knitted to `.markdown`, and this Markdown document is usually rendered to HTML by a site generator named Hugo.
+
+## R Markdown Anatomy {#rmd-anatomy}
+
+We can dig one level deeper by considering the different components of an RMarkdown. Specifically, let's look at when and how these are altered during the rendering workflow. 
+
+### YAML Header
+
+The YAML header is processed in many stages of the rendering process and can influence the final document in many different ways. It is read by each of Pandoc, **rmarkdown**, and **knitr**. Along the way, the information is contains can effect the code, content, and the rendering process. 
+
+A typical YAML header might look something like this, containing basic metadata and rendering instructions:
+
+````md
+```
+title: My R Markdown report
+author: Yihui Xie
+output: html_document
+```
+````
+
+In this case, the `title` and `author` fields are processed by Pandoc to set the values of template variables. Using the default template, the title and author information will appears at the beginning of the resulting document. More details on how Pandoc uses information from the header are explained on the [R Markdown website's introduction to Pandoc Markdown](https://rmarkdown.rstudio.com/authoring_pandoc_markdown.html%23raw-tex#Metadata_blocks). 
```

Comment:
Maybe it is better to link to Pandoc Manual directly for this ? 🤔 
https://pandoc.org/MANUAL.html#extension-yaml_metadata_block

Reply:
Excellent point - thank you! I will switch it over.